### ISSUE TITLE
Quick docs for commands show "No documentation found"

### ISSUE THREAD

**Issue by liurui39660** [ArtsiomCh/CMake#13]
From the release note of 193.3.1, it says that quick docs for some commands like `IF`, `FOREACH`, etc are added. I just updated to the latest release from CLion plugin settings but found that quick docs for almost all commands (not only those newly added) only show "No documentation found".

I also tested the quick doc for other things, they can show the quick docs properly
- Built-in variables, e.g. `CMAKE_CXX_STANDARD`, `PROJECT_NAME`
- Command properties, e.g. `VERSION` in `CMAKE_MINIMUM_REQUIRED`
- Policy, e.g `CMP0091` in `CMAKE_POLICY`

I'm not sure what information might be helpful for identifying the issue, please tell me if you need any.

CLion generated troubleshooting information is as below
I don't know why it says my total memory is 522MiB, I actually have 32GiB in total.

```
=== About ===
Build version: CLion 2019.3.2 Build #CL-193.5662.56 December 18, 2019
Java version: 11.0.5+10-b520.17amd64
Operating System: Windows 10 (10.0, amd64)
JVM version: OpenJDK 64-Bit Server VM JetBrains s.r.o

=== System ===
Number of CPU: 8
Used memory: 394Mb 
Free memory: 128Mb 
Total memory: 522Mb 
Maximum available memory: 4083Mb

=== Displays ===
Displays: 
Display 0: 900x1440; scale: 1.00
Display 1: 1920x1080; scale: 1.00

=== Plugins ===
Custom plugins: [Docker (193.5662.7), GLSL Support (1.19), Grep Console (9.8.192.4205.1), File Watchers (193.5662.31), Properties (193.5662.65), Scratch (1.10), CMake simple highlighter (193.3.1), CMake Plus (193.3.1), CLion Help (193.5662.56), Material Theme UI (4.11.5), Code Notes (1.3), Path hide (1.0), Toml (0.2.111.34-193), Grazie (2019.3-6.2.stable), Rust (0.2.113.2150-193), Custom Postfix Templates (2.10.13.193), Rainbow Brackets (5.29), leetcode editor (6.1), .ignore (3.2.3.193), Makefile support (1.7.0), IDEA Mind Map (1.4.7), Quick File Preview (1.4.8), TeXiFy IDEA (0.6.7), AsciiDoc (0.30.21), Nyan Progress Bar (1.14), GitToolBox (193.3.3)]
Disabled plugins:[]
```

**Comment by ArtsiomCh** [comment]
Hmm... just checked with CL-193.5662.56 and QuickDocs works fine for commands and newly added keywords...
Could you send me your idea.txt file (Help->Show Log in Explorer ), please?

**Comment by liurui39660** [comment]
Sorry for the late reply, I uploaded the `idea.log` to my [Google drive](https://docs.google.com/document/d/1O4yR6c7q39YaBzvykgXqv5sW2lp8DaapLnRh90IFDug/edit?usp=sharing)

I first cleared the existing log file, then did the following steps
- Open CLion
- Choose an existing project from the welcome window (my setting is not to automatically load the last opened project)
- Quick doc on some commands, properties and variables
- Close the project, go back to the welcome window
- Close the welcome window

The timestamp in the log file is my local time, which is UTC+8

I tried to paste the log file here, but it exceeds the length limit

**Comment by ArtsiomCh** [comment]
I don't see anything wrong with my plugin in the log... 
And the worst is that I can't reproduce it with the same version of CLion and Windows OS...
Ok, let me make a special version of CMake Plus plugin with heavy logging for the Quick docs process and then we'll run it on your system and hopefully will see errors in the log.

**Comment by ArtsiomCh** [comment]
@liurui39660 you mentioned:
>  quick docs for almost all commands (not only those newly added) only show "No documentation found".

Could you please give me the example of a command for which quick docs show "No documentation found" and also a command for which you able to get quick docs?

**Comment by ArtsiomCh** [comment]
Got it. It's about case sensitivity. So quick doc for 'set()' will be shown but not for 'Set()' or 'SET()'.
Will fix it tomorrow.

**Comment by liurui39660** [comment]
Sorry for forgetting to put a sample code snippet. But indeed, all the commands I use are in upper case. I just tried to use quick docs on lower-case commands, and they all worked. This confirms your idea. Thanks for your assistance.

**Comment by ArtsiomCh** [comment]
Fixed at 193.3.2 of CMake simple highlighter plugin.
